problem_builder/management/commands/copy_deprecated_course_ids.py

```diff
@@ -0,0 +1,57 @@
+
+import math
+import time
+from django.core.management.base import BaseCommand
+from problem_builder.models import Answer
+
+
+class Command(BaseCommand):
+    """
+    Copy content of the deprecated Answer.course_id column into 
+    Answer.course_key in batches.  The batch size and sleep time between 
+    batches are configurable.
+    """
+    help = 'Copy content of the deprecated Answer.course_id column to Answer.course_key in batches'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--batch-size',
+            help='The size of each batch of records to copy (default: 5000).',
+            type=int,
+            default=5000,
+        )
+        parser.add_argument(
+            '--sleep',
+            help='Number of seconds to sleep before processing the next batch (default: 5).',
+            type=int,
+            default=5
+        )
+
+    def handle(self, *args, **options):
+        batch_size = options['batch_size']
+        sleep_time = options['sleep']
+        max_pk = Answer.objects.order_by('-pk')[0].pk
+        batch_start = 0
```

Comment:
If we have to stop this command and re-start it, we'll want to start at a different ID. We should make this configurable.  Optionally it could look at the lowest PK where the new field is empty?  That may be more clever than necessary.